plugin.js

```diff
@@ -66,7 +67,7 @@ module.exports = AdminModule = {
                 client.debug("PluginAdmin", format("%s: %s, %s (%s)",
                     name, hostmask[name], admin[name], result));
 
-                return admin[name].test(hostmask[name]);
+                return result;
```

Comment:
You have to expand the preview, its making it look like thats all one function but its squishing two together. Full code here:

https://github.com/LordWingZero/tennu-admin/blob/master/plugin.js#L65